Web/starter_project_barbies/pages/blog/blogs.tsx

```diff
@@ -0,0 +1,70 @@
+import { BlogCardWide } from "@/components/blog/BlogCardWide";
+import { Pagination } from "@/components/common/Pagination";
+import { Blog } from "@/types/blog";
+import React from "react";
+import { AiOutlinePlus } from 'react-icons/ai';
+
+interface BlogsProps {
+  blogs: Blog[]
+}
+
+export const Blogs: React.FC<BlogsProps> = ({ blogs }) => {
+  return (
+    <div className='bg-white text-black font-montserrat'>
+
+      {/* Title and search bar */}
+      <div className="flex justify-center pt-20">
+        <div className='grid grid-cols-3 mt-2 w-full'>
+          <div className="flex items-center">
+            <div className='ps-28 text-xl font-black'>
+              Blogs
+            </div>
+          </div>
+          <div className='flex justify-center'>
+            <SearchForm />
+          </div>
+          <div />
+        </div>
+      </div>
+
+      {/* Blog list */}
+      <div className="mt-10 px-56">
+        <BlogsList blogs={blogs} />
+      </div>
+
+      {/* Pagination */}
+      <Pagination numberOfPages={5} />
+
+    </div>
+  )
+}
+
+const BlogsList: React.FC<BlogsProps> = ({ blogs }) => {
+  let blogComponents = []
+  let key = 0
+  for (let blog of blogs) {
+    blogComponents.push(<BlogCardWide blog={blog} key={key++} />)
+  }
+
+  return (
+    <div className='bg-white text-black'>
+      {blogComponents}
+    </div>
+  )
+}
+
+const SearchForm = () => {
+  return (
+    <form className="flex gap-4">
+
+      {/* Search input */}
+      <input type="text" className="border border-gray-300 text-gray-900 text-sm rounded-full block px-8 py-2.5 w-60" placeholder="Search..." />
```

Comment:
It's better to define the color on tailwind config file if you think other will use it too